pages/common/hg-log.md

```diff
@@ -0,0 +1,35 @@
+# hg log
+
+> Display the revision history of the repository.
+
+- Display the entire history of the repository:
```

Comment:
Perhaps `revision history` instead of `history` to keep it consistent would be better?

Reply:
Sounds good 👍